src/components/MySpace/MySpace.stories.tsx

```diff
@@ -18,29 +18,39 @@ export default {
 
 const exampleCollection = [
```

Comment:
nitpick alert: should this be named something other than collection since it contains news now?